src/Components/FormElement.js

```diff
@@ -1,38 +1,65 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Input from './Input'
 export default function FormElement(props) {
+
+  const handleSubmit = async (event)=>{
+    event.preventDefault();
+    const headers = new Headers({
+      'Content-Type': values["Content-Type"],
+      'X-HS-AccessKey': values["X-HS-AccessKey"]
+    })
+    try{
+      await fetch(`http://localhost:8080/v2/partner/subscription/${values["SubscriptionId"]}`, {
+      method: 'GET',
+      headers,
+      mode: 'cors'
+         })
+         .then(results => console.log(results))}
+
+    catch(error){console.error(error)}
+
+  }
+  const [values, setValues] = useState({})
```

Comment:
const {obj:{pathParams, queryParams, headers}={}} = props
....
pathParams && (<></>)

Reply:
add prototypes for each component - https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
added proptypes and destructuring of the object is done.